app/templates/message_template.hbs

```diff
@@ -1,34 +1,36 @@
 <section class="message-section">
   <div class="row">
     <div class="small-12 columns">
-      {{#each groupedElements as |messages|}}
-        <div class="small-12 columns day_seperator">
-          <span>{{display-messages-date messages.firstObject.createdDate (t 'day.today')}}</span>
-        </div>
-        {{#each messages as |message|}}
-          {{#if message.isMessage}}
-            <div class="{{message.state}} {{if message.myMessage 'my_message' 'received_message'}}" id={{message.id}}>
-              <div class='message_details'>
-                {{#if message.myMessage}}
-                  {{t "messages.owner" }}
-                {{else}}
-                  {{custom-img-tag src=message.sender.displayImageUrl class="user-avatar"}}
-                  {{t "full_name" firstName=message.sender.firstName lastName=message.sender.lastName}}
-                  {{message.sender.roleInitials}}
-                {{/if}}
-                →
-                {{display-datetime message.createdAt format="HH:mm"}}
-              </div>
+      {{#href-to-link-to}}
```

Comment:
@swatijadhav I think it would be better to have a handlebars helper that generates the correct route for each link without specifying a specific href. That will make the system more flexible in later iterations if we change routes. E.g. IF the message contains <a href="/offers/12/review">Link</> then it will always point to that place. If we changed it to <a data-route-name="review_offers" data-params="id:12">Link</a> then we can run the link through a generateRoute helper that will output the correct href. That way it will work irrespective of future updates or whether the app is in cordova or browser, or whichever routing strategy ('#' or history) is being used.

This is a note for a future refactor, I don't think we need to do anything right now. Will make a ticket. Feel free to add your thoughts too.